core/src/main/java/org/elasticsearch/search/internal/InternalSearchHit.java

```diff
@@ -200,6 +200,9 @@ public NestedIdentity getNestedIdentity() {
      */
     @Override
     public BytesReference sourceRef() {
+        if (this.source == null)
+            return null;
```

Comment:
could you please add the curly brackets around this statement? we tend to use them although not needed around a single, for better readability.